justfile

```
@@ -28,7 +28,7 @@ build:
 
 # Setup the project to use a pinned version of the CLI
 setup:
-    -cargo binstall -y --install-path ./target/bin stellar-cli --version 22.6.0
+    -cargo install stellar-cli --version 22.6.0 --root ./target
```

```suggestion
    -cargo install stellar-cli --debug --version 22.6.0 --root ./target
```
Will speed up the compile step. And the debug version isn't so slow.